package/gluon-web-model/luasrc/usr/lib/lua/gluon/web/model/classes.lua

```diff
@@ -257,7 +257,7 @@ function AbstractValue:handle()
 	end
 end
 
-function AbstractValue:write(value)
+function AbstractValue:write(_)
```

Comment:
Hmm, as this is supposed to be overridden, let's at least add a comment above. But also drop the unused `_` for consistency.

Reply:
Added a comment and dropped `_`